dashboard/app/models/rubric.rb

```diff
@@ -18,11 +18,16 @@ class Rubric < ApplicationRecord
   belongs_to :lesson
 
   def summarize
+    level_position = lesson.script_levels.filter {|sl| sl.levels.include?(level)}.first&.position
```

Comment:
Open to feedback on how to make this cleaner

Reply:
Oh that's much better thank you!

Reply:
I see you merged already. never mind, not important enough to open a new PR just for this

Reply:
It's in the back of my mind to add some assurances that the level of a rubric exists within the lesson, but we don't have it yet. I'd rather enforce that elsewhere, though. Just created [a task](https://codedotorg.atlassian.net/browse/AITT-148) for that